In [50]:
import numpy as np
import matplotlib.pyplot as plt
import os
import re
from collections import Counter
import json
from itertools import islice
import ast

In [36]:
with open('pos_forms.txt', 'r') as f:
    positive_forms = [linea.strip() for linea in f if linea.strip()]
with open('negative_forms.txt', 'r') as f:
    negative_forms = [linea.strip() for linea in f if linea.strip()]
with open('neutral_forms.txt', 'r') as f:
    neutral_forms = [linea.strip() for linea in f if linea.strip()]

In [37]:
print(len(positive_forms))
print(len(negative_forms))
print(len(neutral_forms))

136
104
104


In [47]:
neu = []
for text in neutral_forms:
    if len(text) < 2:
        print(text)
    else:
        neu.append(text)
print(len(neu))

"
103


In [48]:
with open('neutral.jsonl', 'w') as f:
    json.dump(neu,f)

### Lortutako esaldiak:
* **Positiboa:** 130 esaldi
* **Negatiboa:** 103 esaldi
* **Neutroa:** 103 esaldi

### **CHATGPT**-ren esaldiak

In [52]:
with open('positive.txt', 'r') as f:
    positive = [linea.strip() for linea in f if linea.strip()]
with open('negative.txt', 'r') as f:
    negative = [linea.strip() for linea in f if linea.strip()]
with open('neutral.txt', 'r') as f:
    neutral = [linea.strip() for linea in f if linea.strip()]

In [53]:
print(len(positive))
print(len(negative))
print(len(neutral))

178
180
180


In [55]:
with open('neutral_gpt.jsonl', 'w') as f:
    json.dump(neutral,f)
with open('positive_gpt.jsonl', 'w') as f:
    json.dump(positive,f)
with open('negative_gpt.jsonl', 'w') as f:
    json.dump(negative,f)